blazor/message/customization.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: post
+title: Customization with Blazor Message Component | Syncfusion
+description: Checkout and learn about Customization with Blazor Message component in Blazor Server App and Blazor WebAssembly App.
+platform: Blazor
+control: Message
+documentation: ug
+---
+
+# Customization in Blazor Message
```

Comment:
I have changed the title as **Customization in Blazor Message** to resolve the below ci failure.
https://jenkins-ci.syncfusion.com/job/Content/job/blazor-docs/5201/artifact/cireports/FrontMatterValidation/FrontMatterValidation.html